src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1087,17 +1087,9 @@ internal List<ServiceController> DoStopService(ServiceController serviceControll
         /// True if all dependent services are stopped
         /// False if not all dependent services are stopped
         /// </returns>
-        private bool HaveAllDependentServicesStopped(ICollection<ServiceController> dependentServices)
+        private bool HaveAllDependentServicesStopped(ServiceController[] dependentServices)
```

Comment:
I'm not sure what the possible impact of changing the method signature here would be, but given it's a private method anyway, it's probably safe to do as long as the code calling the method is still OK to call it with this change.

Reply:
This method is only called from one place, hence I believe it's separated to provide better readability.